openid-4-verifiable-presentations-1_0.md

```diff
@@ -2025,19 +2025,102 @@ The Verifier MAY send all the OpenID4VP request parameters as members in the req
 
 The Verifier MAY send a signed request, for example, when identification and authentication of the Verifier is required.
 
+The signed request allows the Wallet to authenticate the Verifier using one or more trust framework(s) other than the Web PKI utilized by the browser. An example of such a trust framework is the Verifier (RP) management infrastructure set up in the context of the eIDAS regulation in the European Union, in which case, the Wallet can no longer rely only on the web origin of the Verifier. This web origin MAY still be used to further strengthen the security of the flow. The external trust framework could, for example, map the Client Identifier to registered web origins.
+
 The signed Request Object MAY contain all the parameters listed in (#browser_api_request), except `request`.
 
-Below is a non-normative example of such a request sent over the W3C Digital Credentials API:
+Verifiers can format signed Requests either using JWS Compact Serialization or JWS Serialization [@!RFC7515]). 
+
+#### JWS Compact Serialization
+
+In case of the JWS Compact Serialization, all request parameters are encoded in a request object as defined in (#vp_token_request) and the JWS object is used as value of the `request` claim in the `data` element of the API call. This is illustated in the following example.
 
 ```js
-{ request: "eyJhbGciOiJF..." }
+const credential = await navigator.credentials.get({
+  digital: {
+    providers: [{
+      protocol: "openid4vp",
+      data: {
+        { request: "eyJhbGciOiJF..." }
+     }
+    }]
+  }
+});
 ```
 
-This is an example of the payload of a signed OpenID4VP request used with the W3C Digital Credentials API:
+This is an example of the payload of a signed OpenID4VP request used with the W3C Digital Credentials API in conjunction with JWS Compact Serialization:
+
+<{{examples/digital_credentials_api/signed_request_payload_compact.json}} 
+
+#### JWS JSON Serialization
+
+The JWS JSON Serialization [@!RFC7515]) allows the Verifier to use multiple Client Identifiers and corresponding key material and metadata for the same request. This is to serve use cases where the Verifier requests credentials belonging to different trust frameworks and thus must authenticate in the context of those trust frameworks.
+
+In this case, the following request parameters MUST be present in the protected header of the respective signature object: 
+
+* `client_id`
+* `client_metadata`
```

Comment:
Since these become protected headers, do we need an IANA registration for this @selfissued ?

Reply:
Yes, if you're going to create new header parameters, you should register them.

Reply:
added IANA registration

**https://github.com/openid/OpenID4VP/pull/308/commits/e2fe62eea5b458e1cd6e402319a3db9db68728e7**